e2e_test/nexmark/create_sources.slt.part

```diff
@@ -29,59 +29,14 @@ CREATE SOURCE nexmark (
 ) WITH (
     connector = 'nexmark',
     nexmark.split.num = '2',
```

Comment:
let me try increase this to see if can reduce sleep time

Reply:
`nexmark.min.event.gap.in.ns = '100'` already,
so actually makes little sense why need to increase sleep number